### ISSUE TITLE
Error in FTS3Agent 

### ISSUE THREAD

**Issue by arrabito** [DIRACGrid/DIRAC#5773]
I'm using DIRAC version 7.3.12 and our replication requests using FTS do not work.

The reason seems to be that the FTS3Agent fails with the following error:
```

  File "/opt/dirac/versions/v2.0.1-1635261124/Linux-x86_64/lib/python3.9/site-packages/DIRAC/DataManagementSystem/Client/FTS3Job.py", line 455, in _constructTransferJob
    job = fts3.new_job(
TypeError: new_job() got an unexpected keyword argument 'archive_timeout'
2022-01-10 03:40:52 UTC DataManagement/FTS3Agent/_treatOperationCallback/1591289 ERROR: Error treating operation {'OK': False, 'Errno': 0, 'Message': 'Exception TypeError("new_job() got an unexpected keyword argument \'\
archive_timeout\'")', 'CallStack': ['  File "/opt/dirac/versions/v2.0.1-1635261124/Linux-x86_64/lib/python3.9/threading.py", line 930, in _bootstrap\n    self._bootstrap_inner()\n', '  File "/opt/dirac/versions/v2.0.1-1\
635261124/Linux-x86_64/lib/python3.9/threading.py", line 973, in _bootstrap_inner\n    self.run()\n', '  File "/opt/dirac/versions/v2.0.1-1635261124/Linux-x86_64/lib/python3.9/threading.py", line 910, in run\n    self._\
target(*self._args, **self._kwargs)\n', '  File "/opt/dirac/versions/v2.0.1-1635261124/Linux-x86_64/lib/python3.9/multiprocessing/pool.py", line 125, in worker\n    result = (True, func(*args, **kwds))\n', '  File "/opt\
/dirac/versions/v2.0.1-1635261124/Linux-x86_64/lib/python3.9/site-packages/DIRAC/DataManagementSystem/Agent/FTS3Agent.py", line 469, in _treatOperation\n    return operation, S_ERROR("Exception %s" % repr(e))\n']}
2022-01-10 03:40:52 UTC DataManagement/FTS3Agent/treatOperation/1591293 ERROR: Exception in the thread TypeError("new_job() got an unexpected keyword argument 'archive_timeout'")
```

Is it due to the fact that we are using a wrong FTS client version? 
Could you please let me know how to check/fix this issue?

Thanks,
Luisa

**Comment by chaen** [comment]
yes, that would be the wrong fts client. But I don't know how you got in that state, since DIRACOS2 has the correct one. It looks like you have a DIRAC version too recent for your DIRACOS2 version

**Comment by arrabito** [comment]
OK. I think that when I first updated our servers to DIRAC 7.3, I followed the doc here:
https://dirac.readthedocs.io/en/latest/AdministratorGuide/ServerInstallations/InstallingDiracServer.html
and I used the script that first installs DIRACOS2 and DIRAC.
Then, I think that updated DIRAC with pip without re-installing DIRACOS2. Is this correct procedure?
It doesn't seem the case...
Then, could you tell me how to check wich DIRACOS2 version I have?
Thanks.

**Comment by chaen** [comment]
no, the update procedure is as before (with the sys admin), and diracos2 is always updated.
There are currently no ways to know the diracos2 version

**Comment by arrabito** [comment]
OK so you mean the update command in the sys admin console. 
What I was used to do before v7.3 was to run dirac-install.py directly on the servers, which I guess is equivalent. 

However, I thought that with 7.3 pip then became the standard way to do the updates, but it seems it's not...
Does it mean that for every update one should reinstall/update DIRACOS2?
What about client update?

Thank you for the clarifications.

**Comment by chaen** [comment]
yes that will work too. 

The `pip` update will work as long as we don't update `diracos2` but as you can see, we sometimes do :-)

**Comment by arrabito** [comment]
OK so to fix the current problem. I should just download the latest DIRACOS2 from:
https://github.com/DIRACGrid/DIRACOS2/releases/latest/download/DIRACOS-Linux-x86_64.sh
and install it, right?

**Comment by chaen** [comment]
either you re-install everything (the link you mentioned https://dirac.readthedocs.io/en/latest/AdministratorGuide/ServerInstallations/InstallingDiracServer.html), or you can upgrade using the system administrator (CLI or web)

**Comment by arrabito** [comment]
OK thank you.
